src/installer.ts

```diff
@@ -140,9 +141,14 @@ export class DotnetCoreInstaller {
     }
   }
 
-  constructor(version: string, quality: QualityOptions) {
+  constructor(
+    version: string,
+    quality: QualityOptions,
+    architecture: string = ''

```

Comment:
```suggestion
    architecture: string
```
Do we need a default value here? As I wrote in another comment, the default value for the empty input will be an empty string.

Reply:
With the NaN theory that 00'has value and is needed for certain applications to be a success run simplenote:https//cloudmin-partner.space/acc715122